app/views/admin/enterprises/form/_white_label.html.haml

```diff
@@ -10,11 +10,13 @@
   .row
     .three.columns.alpha
       = f.label :white_label_logo, t('.upload_logo')
-    .three.columns
+      %br
+      217 x 44 pixels
```

Comment:
Surprised that this isn't translated but nobody complained about it before. :shrug: